src/auth0-sso-login.js

```diff
@@ -31,8 +31,10 @@ export class localStorageInteraction {
   }
 }
 
+export const tokenExpiresAtKey = 'cimpress.auth0.tokenExpiresAt';
```

Comment:
Should we call this to match the library? `cimpress.auth0-sso-login.tokenExpiresAt`

Another question why export this via a function as in `getTokenExpiry`, shouldn't we hide the usage of the `localStorage` to prevent the exposing the implementation details and allowing clients to alter this value directly.

Reply:
For other reviewers: Not relevant anymore, use of `localStorage` was dropped.